docs/bin/plugin_formatter.py

```diff
@@ -183,8 +184,8 @@ def write_data(text, output_dir, outputname, module=None):
             os.makedirs(output_dir)
         fname = os.path.join(output_dir, outputname)
         fname = fname.replace(".py", "")
-        with open(fname, 'wb') as f:
-            f.write(to_bytes(text))
+
+        update_file_if_different(fname, text)
```

Comment:
You need either `to_bytes(text)` or `text.encode('utf-8')` here as well.

Reply:
Fixed.